### PR TITLE
Dev lab2

### PR DESCRIPTION
**Pull request by IHavePortableBrain**
Vladislav Krestinin 751005 lab2 adaptive layout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                <li>
+                    <a href="">BLOG</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <div class="banner">
+        <a class="ref" href="http://blazrobar.com/">
+            <span class="content">
+                <span class="info">
```
Comment (aliaksandrkrasnou) [inline index.html:42]: Semantically it's better to use h1 tag

File: style.css [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    color: #37a0b4;
+    width: 100%;
+    box-sizing: border-box;
+    font-weight: bold;
+    padding-top: 2em;
+    padding-bottom: 2em;
+}
+
+main .link,
+main article{
```
Comment (aliaksandrkrasnou) [inline style.css:220]: To read: Yandex BEM

**Comment by aliaksandrkrasnou** [comment]
I noticed that you are from 751005 group, I'm not your teacher

**Comment by IHavePortableBrain** [comment]
Sorry, i have used link to repo of yours from common task.

**Pull request closed without merging**